components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -1123,12 +1131,15 @@ private String populateOauthParameters(OAuth2Parameters params, OAuthMessage oAu
             OIDC Request object will supersede parameters sent in the OAuth Authorization request. So handling the
             OIDC Request object needs to done after processing all request parameters.
          */
-        try {
-            handleOIDCRequestObject(oauthRequest, params);
-        } catch (RequestObjectException e) {
-            // All the error logs are specified at the time when throw the exception.
-            return EndpointUtil.getErrorPageURL(e.getErrorCode(), e.getErrorMessage(), null);
+        if (oauthRequest.getScopes().contains(OAuthConstants.Scope.OPENID)) {
```

Comment:
If there are no scopes associate with the oauthRequest , oauthRequest.getScopes() can be null. Shall we handle null here?